src/main/java/nextstep/courses/domain/Course.java

```diff
@@ -28,6 +33,15 @@ public Course(Long id, String title, Long creatorId, LocalDateTime createdAt, Lo
         this.updatedAt = updatedAt;
     }
 
+    public Course(Long id, String title, Long creatorId, List<Session> sessions, LocalDateTime createdAt, LocalDateTime updatedAt) {
```

Comment:
```suggestion
    public Course(Long id, String title, Long creatorId, LocalDateTime createdAt, LocalDateTime updatedAt, List<Session> sessions) {
```

앞의 생성자와 인자 순서를 맞추고 마지막에 인자를 추가하는 것은 어떨까?